src/cli.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+import fs from 'fs'
+import cli from 'args'
+
+import CONS from './constants'
+import getApiCredentials from './get-api-credentials'
+import PriceCsvParser from './main'
+
+process.title = 'csvparserprice'
+
+const args = cli
+  .option(
+    ['i', 'inputFile'],
+    'File to get CSV from.',
+    process.stdin,
+    (arg) => {
+      if (typeof arg === 'string')
+        return fs.createReadStream(String(arg))
+
+      return arg
+    }
+  )
+  .option(
+    ['o', 'outputFile'],
+    'File to output JSON to.',
+    process.stdout,
+    (arg) => {
+      if (typeof arg === 'string')
+        return fs.createWriteStream(String(arg))
+
+      return arg
+    }
+  )
+  .option(
+    'delimiter',
+    'The delimiter that is used in the csv.',
+    CONS.standards.delimiter
```

Comment:
1. Separating it from the constants that are used in the CSV, like the money regex and header names seems better to me... thoughts?

2. Shortening it to 'DELIM' is pretty confusing to me.